client/src/features/signup/components/LoginInfoForm/__snapshots__/LoginInfoForm.test.js.snap

```diff
@@ -0,0 +1,191 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`LoginInfoForm snapshot test should LoginInfoForm match snapshot 1`] = `
+<DocumentFragment>
+  <p>
+    Login Info
+  </p>
+  <div>
+    <div>
+      <div
+        class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-wb57ya-MuiFormControl-root-MuiTextField-root"
```

Comment:
I suppose you've missed mocking components

Reply:
nope.  that is the formik-mui input field.  jest understands it and no conflicts occur in testing if left unmocked.